.circleci/config.yml

```diff
@@ -122,7 +122,7 @@ jobs:
       - run:
           name: Run Test App with network simulator
           # Dropping packets from both sides of the stream to test dealing with lost packets
-          command: SCENARIO="droplist --delay=15ms --bandwidth=10Mbps --queue=25 --drops_to_server=2,5,7 --drops_to_client=6,8" docker-compose -f utp-testing/docker/docker-compose.yml up -d
+          command: SCENARIO="droplist --delay=15ms --bandwidth=10Mbps --queue=25 --drops_to_server=3,5,7 --drops_to_client=6,8" docker-compose -f utp-testing/docker/docker-compose.yml up -d
```

Comment:
note: this change was made because the 2nd packet corresponds to the node's response to the discovery v5 `WHOAREYOU` challenge

Reply:
at this time I'm not sure, but I can look into it further.

Reply:
Any luck tracking this down?

Reply:
unfortunately I have not been able to track this down. I'm not sure why things were working before. that said, I do not think this should block merge because we move the drop into the uTP data transfer, which is the aspect we are interested in testing.